src/routes/coingeckoProxy/index.ts

```diff
@@ -9,9 +9,13 @@ const coingeckoProxy: FastifyPluginAsync = async (
   opts
 ): Promise<void> => {
   fastify.register(httpProxy, {
-    upstream: "https://api.coingecko.com",
-    rewritePrefix: "/api/v3/simple/",
+    upstream: "https://pro-api.coingecko.com",
```

Comment:
Our key has been temporarily set to "pro" for testing.
For this reason, we must use the pro api endpoint.

I'll switch this to and env var so we don't need a deployment to change it.